# Supplementary Material

# Supplementary Figures

## 1- Geographical locations
##### Fig.S 1.4a | Labels of all the groups included in the AfricanNeo dataset. 
https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_Suppl_Material/main/Geographical_Locations/Fig_S_1_4a_Map.html

##### Fig.S 1.4b | Labels of all the populations included in the AfricanNeo dataset. 
https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_Suppl_Material/main/Geographical_Locations/Fig_S_1_4a_Map.html

## 2- Dimensionality reduction methods (DRM)

##### Fig.S 2.1 | UMAP approach on the basis of genotype data. 
https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_Suppl_Material/main/

##### Fig.S 2.2 | PCA plots for only Bantu-speaking populations. 
https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_Suppl_Material/main/

##### Fig.S 2.3 | PCA plots for selected sub-Saharan African populations.
https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_Suppl_Material/main/

##### Fig.S 2.4 | Procrustes rotated PCA for the Only-African dataset. 
https://raw.githack.com/Schlebusch-lab/Expansion_of_BSP_Suppl_Material/main/

## 4- Ancestry-specific analyses

##### Fig.S 4.1 | Ancestry-specific (AS-)PCA for the masked Only-BSP dataset.

##### Legends and maps for each group (top) and each population (bottom) in Fig.S 4.4.


## 5- Genome-wide runs of homozygosity (ROH) estimates

## 6- Estimated effective population sizes (Ne)

## 8- Patterns of haplotype diversity


